base/gcutils.jl

```diff
@@ -10,6 +10,10 @@
 Register a function `f(x)` to be called when there are no program-accessible references to
 `x`, and return `x`. The type of `x` must be a `mutable struct`, otherwise the behavior of
 this function is unpredictable.
+
+`f` must not cause a task switch, which excludes most I/O operations such as `println`.
+`@schedule println("message")` or `ccall(:jl_, Void, (Any,), "message")` may be helpful for
```

Comment:
It looks like `@schedule` is deprecated now, so the hint should probably be to use `@async`